### DISCUSSION TITLE
Made update for plugin SuperSlime

### DISCUSSION THREAD

**Discussion by comdar16** [Pryaxis/TShock#2462] [Show and tell]
[redacted] here is updated plugin SuperSlime

:D

**Comment by moisterrific** [comment]
Thanks for updating that and sharing it with everyone, but for future reference please follow the appropriate plugin submission procedure on the [Official Pryaxis Plugins repository](https://github.com/Pryaxis/Plugins#how-to-submit-plugins) and open a pull request for plugins there, instead of directly linking any releases here. There is no security guarantee whatsoever when you (or anyone really) directly link stuff to `.dll` release files, because they haven't been de-compiled to check for compliance or potential issues. 

Once again I appreciate your effort in updating old plugins that the original devs abandoned but please understand sharing them this way does not inspire trust from TShock users who might be looking for plugins. 

For now I will be removing all these links and locking these discussions, if you have any concerns or questions feel free to ask us on the [Pryaxis Discord](https://discord.gg/Cav9nYX).
